From 6b721f5e1dbdc3b4f91fe36d11c62178c0b4633d Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Mon, 4 May 2026 19:29:00 +0800
Subject: [PATCH] dts: arm: ast2705-evb: enable XGMAC

Enable the XGMAC node on the AST2705 EVB so U-Boot can expose the
10G MAC defined by the AST2705 SoC device tree.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Ia0142719d1f75a536e0bac0ab73fe08602c3f4ab
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 arch/arm/dts/ast2705-evb.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/dts/ast2705-evb.dts b/arch/arm/dts/ast2705-evb.dts
index 16f53f854c..eacc517325 100644
--- a/arch/arm/dts/ast2705-evb.dts
+++ b/arch/arm/dts/ast2705-evb.dts
@@ -75,6 +75,10 @@
 	tx-internal-delay-ps = <0>;
 };
 
+&xgmac {
+	status = "okay";
+};
+
 &fmc {
 	status = "okay";
 	pinctrl-names = "default";
-- 
2.43.0

